### PR TITLE
Update to node 16 and test node 14 in the CI

### PR DESCRIPTION
**Pull request by umbobabo**

### CODE REVIEW

**Review by fenglish — DISMISSED** [review]

**Review by fenglish — COMMENTED** [review]
It doesn't need to remove `package-lock.json` from `.gitignore` file and update?

**Review by fenglish — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -57,7 +61,7 @@ references:
     branches:
       ignore: /.*/
 
-version: 2
+version: 2.1
```
Comment (fenglish) [inline .circleci/config.yml:64]: This one doesn't need `orb`?

Comment (fenglish) [reply .circleci/config.yml:64]:
```
orbs:
  node: circleci/node@4.6.0
```
and 
```
      - node/install-npm:
          version: "7"
```

Comment (umbobabo) [reply .circleci/config.yml:64]: I think the orb was needed as part of the npm7 migration related to Origami. This package doesn't depend on Origami so it seems fine like this

Comment (umbobabo) [reply .circleci/config.yml:64]: However it uses the old shared helper for peer dependencies so I can update it anyway, let me open another PR

**Pull request merged**